include/mesh/exodusII_io_helper.h

```diff
@@ -501,36 +509,42 @@ class ExodusII_IO_Helper : public ParallelObject
   // General error flag
   int ex_err;
 
-  // Number of dimensions in the mesh
-  int num_dim;
+  // struct which contains data fields from the Exodus file header
+  ExodusHeaderInfo header_info;
 
-  // Number of global variables
-  int num_global_vars;
+  // Problem title (Use vector<char> to emulate a char *)
+  std::vector<char> & title;
 
-  // Number of sideset variables
-  int num_sideset_vars;
+  // Number of dimensions in the mesh
+  int & num_dim;
```

Comment:
These are basically backwards compatibility shims to the ExodusHeaderInfo entries?  Do we need them?  Was anyone ever using ExodusII_IO_Helper directly?

I'm in favor of the shims either way, but maybe we could wrap them in an ```#ifdef LIBMESH_ENABLE_DEPRECATED``` and so hopefully get rid of them in another year?

Reply:
We don't *need* them but it made the current diff much smaller, and I think I like the code better with them (easier to read and less typing) but if you really want to we could get rid of them.

Reply:
Hmm... I guess I'm paying attention to the visible redundant lines here and not to the hypothetical redundant tokens you'd need to add elsewhere otherwise.  Okay, I'm happy with it.